### ISSUE TITLE
Basin Masks

### ISSUE THREAD

**Issue by willtebbutt** [JuliaGeo/GeoDatasets.jl#20]
Would the maintainers be interested in a PR adding this ocean basin mask data set?

https://iridl.ldeo.columbia.edu/SOURCES/.NOAA/.NODC/.WOA09/.Masks/.basin/index.html?Set-Language=en

Alternatively, if there's a better mask data source (this is my first time doing this particular bit of work, so I could well have missed whatever it is that is usually used) also happy to work with that!

**Comment by Alexander-Barth** [comment]
Yes, I think it would be certainly useful. Which format do you plan to use?

For your information, there is also the IHO dataset (as shapefile) under the CC BY-NC-SA 4.0 license:

https://marineregions.org/downloads.php#iho

One could also query the web-service. For example this is the North Sea:
http://marineregions.org/mrgid/2350

Here is a [JSON files](
https://geo.vliz.be/geoserver/wfs?request=getfeature&service=wfs&version=1.1.0&typename=MarineRegions:iho&outputformat=json&filter=%3CPropertyIsEqualTo%3E%3CPropertyName%3Eid%3C%2FPropertyName%3E%3CLiteral%3E4%3C%2FLiteral%3E%3C%2FPropertyIsEqualTo%3E) with the boundaries.

**Comment by willtebbutt** [comment]
> Yes, I think it would be certainly useful. Which format do you plan to use?

I was imagining NetCDF, but now that I think about it, I'm not sure how that would play with this package unless you're willing to accept something like `NCDatasets.jl` as a dependency. Do you have a preference?

> One could also query the web-service. For example this is the North Sea:

Ooo nice. Maybe this is a better option?

**Comment by Alexander-Barth** [comment]
I think both data sources can be useful. IHO dataset has a higher resolution, but it is also more complex to work with. Feel free to pick to one who suits your application best :-)

**Comment by willtebbutt** [comment]
Excellent. What are your thoughts on an NCDatasets dependency?

**Comment by Alexander-Barth** [comment]
That is fine. It is very difficult to avoid a dependency. (I tried to load the binary data for the gshhg datasets but I gave up and loaded it via the `Shapefile`).
